### ISSUE TITLE
Question!

### ISSUE THREAD

**Issue by traveldesign** [Lothrazar/BlockyDoors#1]
Minecraft Version: 1.16.5

Forge Version: 

Mod Version: 

Single Player or Server: Single

Describe problem (what you were doing; what happened; what should have happened):

I don't have a problem, but a question. I didn't see anywhere else to write it. 

I love the mod. I'm wondering if you're considering doing trapdoors as well? Thanks!

Log file link:

Video/images/gifs (direct upload or link):

**Comment by Lothrazar** [comment]
yeah! maybe trapdoors in the 1.18 version? i already have trapdoors in 1.16 in this other mod https://www.curseforge.com/minecraft/mc-mods/absent-by-design
